### PR TITLE
refactor(page_objects): FRMW-742 remove deprecated and aliased methods

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-742

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] QE LGTM

---
Refactored using new `utils/rx-page-objects` construct.

### CODE REVIEW

**Comment by 100stacks** [comment]
@Droogans, my original PR removed `rxFloatingHeader`: https://github.com/rackerlabs/encore-ui/blob/master/utils/rx-page-objects/src/rxFloatingHeader.page.js -- since it includes only _**aliased**_ methods.  

Please let me know if you want to keep `rxFloatingHeader.page.js` or remove it in a future iteration.

**Comment by 100stacks** [comment]
Updated `2.0.0-8 Release Notes (unreleased)`:
https://github.com/rackerlabs/encore-ui/releases/tag/untagged-40b13b67496b25ef9aab

Updated `2.0 Migration Guide`:
https://github.com/rackerlabs/encore-ui/wiki/2.0-Migration-Guide#v200-8-eliminate-deprecatedaliased-page-objects

**Comment by CITguy** [comment]
LGTM

File: utils/rx-page-objects/test/rxOptionTable.midway.js [hunk 1]
```diff
@@ -113,8 +113,8 @@ describe('rxOptionTable', function () {
             expect(component.selections).to.eventually.eql([0, 1]);
         });
 
-        it('should unselect many rows', function () {
-            component.unselectMany([{ Name: 'Item 1' }, { Name: 'Item 2' }]);
+        it('should deselect many rows', function () {
```
Comment (CITguy) [inline utils/rx-page-objects/test/rxOptionTable.midway.js]: 👍

**Comment by kevincareer427** [comment]
LGTM

**Pull request merged**